src/helpers/validation.ts

```diff
@@ -0,0 +1,36 @@
+import Ajv from 'ajv';
+import { isAddress } from '@ethersproject/address';
+
+export function validateForm(schema, form) {
+  const ajv = new Ajv({ allErrors: true });
+
+  ajv.addFormat('address', {
+    validate: isAddress
+  });
+
+  ajv.addFormat('uint256', {
+    validate: value => !!value.match(/^[0-9]+$/)
```

Comment:
There is a constant on ethers.js with the max value you can get, we could use this for validation https://docs.ethers.io/v5/api/utils/constants/#constants-MaxUint256, we could check if number is between 0 and max instead of using string filtering, this would enforce number to not start with zero(s) example: 00012345.
We could also add a format for `int256` which is same as `uint256` expect that it can go to negative there is also constants for the limits see: https://github.com/ethers-io/ethers.js/blob/master/packages/constants/src.ts/bignumbers.ts#L10-L11 you will need to update here too to add new format https://github.com/snapshot-labs/sx-ui/blob/db27536ad7ed659990d0c044ce2dc880f2e593a8/src/helpers/utils.ts#L115-L118